health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -481,6 +481,47 @@ const createUpdatePlanProject = async (req) => {
+                return updatedRow;
+              })
+            : [];
+
+        //get the list of assumptions from UI
+        const formulasToUpdateFromUIForSubFormula = req?.formulasToUpdate?.filter((row) => {
+          return row.category && row.output && row.input && row.operatorName && row.assumptionValue;
+        })
+        ?.map((row) => {
+          const updatedRow = { ...row };
+          const operatorName = row?.operatorName;
+          delete updatedRow?.operatorName;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)